signing/src/main/java/tech/pegasys/web3signer/signing/config/KeystoresParameters.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2022 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.signing.config;
+
+import java.nio.file.Path;
+
+public interface KeystoresParameters {
+
+  Path getKeystoresPath();
+
+  Path getKeystoresPasswordsPath();
+
+  Path getKeystoresPasswordFile();
```

Comment:
Since teku compatibility is a goal of this PR, I'm wondering if there's a further benefit to making it even closer to how this works https://docs.teku.consensys.net/en/latest/Reference/CLI/CLI-Syntax/#validator-keys

i.e. --validator-keys="<keystoresPath>:<keystoresPasswordsPath>"
The naming and similar structure may be easier for users to translate if they're already used to teku.

Reply:
I don't think needs to exactly the same as the Teku implementation just compatible and inspired by Teku as in we can pass in a directory of keystores and keystore passwords. The feature should also make sense in how Web3Signer works as well.

The approach I've taken follows the bulk loading approach we have done elsewhere. Combining the bulk loading with individual key configuration doesn't make sense for web3signer. We have the existing config files for this kind of configuration. So I only included the directory part.

As far as command line options go we unfortunately cannot have something as generic as what done with Teku as we have multiple key sources so --validator-keys is not clear for web3signer what source they are coming from which why have taken the same naming approach which we took with Azure bulk loading.